src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
```

Comment:
In the 'getById' function, you are using the Unary Plus (+) operator to convert the 'id' parameter to a number. While this is not necessarily an issue, it could lead to unexpected results if a non-numeric string is passed as 'id'. This could lead to a bug in your application. Consider adding error handling to check if the 'id' is a valid number before using it.